src/utils/use-title.ts

```diff
@@ -0,0 +1,47 @@
+import { useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+
+const TRUNCATE_LENGTH = 10
+
+export const setTitle = (title: string) => {
+  document.title = title
+}
+
+export const useTitle = (pagePrefix?: string) => {
+  const urlParams = useParams()
+
+  useEffect(() => {
+    const currentTitle = document.title
+    const pageTitleParams: string[] = []
+    if (urlParams.transactionId) {
+      pageTitleParams.push(`Txn:${urlParams.transactionId.slice(0, TRUNCATE_LENGTH)}`)
```

Comment:
```suggestion
      pageTitleParams.push(`Txn:${ellipseId(urlParams.transactionId)}`)
```
This will inherit the same convention as we use elsewhere.

Reply:
👍 made the change